presto-benchto-benchmarks/pom.xml

```diff
@@ -26,6 +26,12 @@
             <groupId>com.teradata.benchto</groupId>
             <artifactId>benchto-driver</artifactId>
             <version>0.4</version>
+            <exclusions>
+                <exclusion>
```

Comment:
Please add a comment explaining why we need to do this

Reply:
Hi @tdcmeehan ,
For benchto-driver - version 0.4 is the latest I see in the maven repository but it has a child dependency spring-web which in turn has a child dependency spring-core version 4.1.6.RELEASE which has the 14 CVE reported, so I have excluded that dependency to resolve it.

Reply:
Oh yes I understand, I mean please add a little comment in the POM file explaining this, since we don't directly use this dependency in the project.

Reply:
Sorry, I have misunderstood, I have added comments in the POM file for these changes, please review